<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointsRectificationWidget</class>
 <widget class="QWidget" name="PointsRectificationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Points Selector</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/main.qrc">
    <normaloff>:/new/graph/draw_points.png</normaloff>:/new/graph/draw_points.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="PaintImageWidget" name="leftWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>400</height>
           </size>
          </property>
         </widget>
         <widget class="PaintImageWidget" name="rightWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="1" rowspan="7">
        <widget class="QTableView" name="pointsMatchingTableView"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="lockButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Lock Zoom</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/main.qrc">
           <normaloff>:/new/prefix1/lock.png</normaloff>
           <normalon>:/new/prefix1/unlock.png</normalon>:/new/prefix1/lock.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="addButton">
         <property name="styleSheet">
          <string notr="true">Text-align:left</string>
         </property>
         <property name="text">
          <string>Add pair</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/main.qrc">
           <normaloff>:/new/prefix1/plus.png</normaloff>:/new/prefix1/plus.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="deleteButton">
         <property name="styleSheet">
          <string notr="true">Text-align:left</string>
         </property>
         <property name="text">
          <string>Delete pair</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/main.qrc">
           <normaloff>:/new/prefix1/minus.png</normaloff>:/new/prefix1/minus.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="rectifyButton">
         <property name="styleSheet">
          <string notr="true">Text-align:left</string>
         </property>
         <property name="text">
          <string>Rectify</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/main.qrc">
           <normaloff>:/new/prefix1/georectify.png</normaloff>:/new/prefix1/georectify.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="7">
        <widget class="QTableWidget" name="pointsMatchingTableWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintImageWidget</class>
   <extends>QWidget</extends>
   <header>uis/paintImageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/main.qrc"/>
 </resources>
 <connections/>
</ui>
